src/services/temporal/core.service.ts

```
@@ -0,0 +1,20 @@
+import { Client, Connection } from '@temporalio/client';
+import config from '../../config';
+
+export class TemporalCoreService {
+  private connection: Connection | undefined
+  private client: Client | undefined
+
+  constructor() { }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)